avm/res/fabric/capacity/main.bicep

```
@@ -34,6 +34,9 @@ param skuTier string = 'Fabric'
 @description('Required. List of admin members. Format: ["something@domain.com"].')
 param adminMembers array
 
+@description('Optional. The lock settings of the service.')
+param lock lockType
```

Hello @hundredacres,
This was not yet communicated broadly, but we now have the chance to reduce code duplication for common UDT interfaces, such as locks, by importing them from the recently released [avm/utl/types/avm-common-types](https://github.com/Azure/bicep-registry-modules/tree/main/avm/utl/types/avm-common-types) utility module. 

This would be as easy as the following:

1.  Replace this block with the following code
```suggestion
import { lockType } from 'br/public:avm/utl/types/avm-common-types:0.2.1'
@description('Optional. The lock settings of the service.')
param lock lockType?
```
2. Remove the Definition at the bottom of this file. That is not required anymore thanks to the import.
3. Regenerate the module via the usual Set-AVMModule utility

An example of where that is already implemented: https://github.com/Azure/bicep-registry-modules/blob/main/avm/res/storage/storage-account/main.bicep#L145

---

Updated